src/datasets/batch_view/longitudinal/reference.md

```diff
@@ -50,6 +50,6 @@ The DAG is [here](https://github.com/mozilla/telemetry-airflow/blob/54cffc42a2ca
 # Code Reference
 
 This dataset is generated by
-[telemetry-batch-view](https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/Longitudinal.scala).
+[telemetry-batch-view](https://github.com/mozilla/telemetry-batch-view/blob/master/GRAVEYARD.md#longitudinal).
```

Comment:
This dataset is now obsolete, isn't it? Can you quickly move this to obsolete and add a note?